chemprop/v2/cli/predict.py

```diff
@@ -39,160 +40,156 @@ def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
 
     @classmethod
     def func(cls, args: Namespace):
-        process_args(args)
-        validate_args(args)
-        main(args) 
+        args = process_common_args(args)
+        validate_common_args(args)
+        args = process_predict_args(args)
+        validate_predict_args(args)
+        main(args)
+
 
 def add_predict_args(parser: ArgumentParser) -> ArgumentParser:
     parser.add_argument(
         "-i",
-        "--input",
         "--test-path",
-        dest="test_path",
         type=str,
-        help="Path to CSV file containing SMILES for which predictions will be made.",
+        required=True,
+        help="Path to an input CSV file containing SMILES.",
     )
     parser.add_argument(
-        "-o", 
-        "--output-dir",
+        "-o",
+        "--output",
         "--preds-path",
-        dest="preds_path",
         type=str,
-        help="Path to CSV or PICKLE file where predictions will be saved.",
+        required=True,
+        help="Path to CSV or PICKLE file where predictions will be saved. If the file extension is .pkl, will be saved as a PICKLE file. If not provided and the test_path is /path/to/test/test.csv, predictions will be saved to /path/to/test/test_preds.csv.",
```

Comment:
It doesn't make sense to require the user to give this argument and also tell them in the help that we have a default if they don't provide it. One of those should be removed.